src/radio/utils/MapUtils.js

```diff
@@ -0,0 +1,10 @@
+/* @flow */
+
+export function getWithDefault<K, V>(map: Map<K, V>, key: K, defaultValue: V): V {
```

Comment:
Even after reading the radio changes, I'm not entirely sure what this is for.

Reply:
The old code was:
```js
if (!this.queues.has(user.id)) this.queues.set(user.id, []);
const q = this.queues.get(user.id) || []; // <- basically a $FlowFixMe-tier hack
```

This was Grade A Spooky Shit:tm: because we want to mutate the array returned by `this.queues.get(user.id)` (which in itself is spooky but that's maybe a refactor for another time).

Because we set it immediately before getting it, we can logically assert that it will never be empty. It is always an array.

Flow can't figure that out, so we had two options:
1. `$FlowFixMe` literally every time we use `q`. This is ugly and terrible.
2. Ensure Flow knows it's always an array by providing a backup value in case it is empty. It is never empty, but the presence of the backup value masks the idea that we definitely want to mutate the original array.
3. <sub>there's more but they become increasingly more arcane</sub>

Instead, I move it to a helper. From the types, we can see that you pass in a `Map<K, V>` and it *always* returns something of type `V`.

This condenses all the logic and Flow-appeasement of the earlier snippet into:
```js
const q = getWithDefault(this.queues, user.id, []);
```